### PR TITLE
Add error when scratchdb is down

### PR DESCRIPTION
**Pull request by RedGuy12**
continue #1789

### CODE REVIEW

File: addons/scratchstats/userscript.js [hunk 1]
```diff
@@ -39,90 +39,92 @@ export default async function ({ addon, msg, console }) {
...
-              label: msg("followers-label"),
-              data: historyData.map((item) => {
-                return { x: Date.parse(item.date), y: item.value };
-              }),
-              fill: false,
-              showLine: true,
-              borderColor: "#4d97ff",
-              lineTension: 0,
+  fetch(`https://scratchdb.lefty.one/v3/user/info/${username}`)
+    .then(async function (response) {
```
Comment (WorldLanguages) [inline addons/scratchstats/userscript.js:43]: Can you keep this as a single line to prevent all lines below being linted again?

Comment (RedGuy12) [reply addons/scratchstats/userscript.js:43]: oop i think vscode formatted it without permission

Comment (RedGuy12) [reply addons/scratchstats/userscript.js:43]: wait no it's the ci that's formatting it like that...

Comment (WorldLanguages) [reply addons/scratchstats/userscript.js:43]: Huh weird, we haven't changed our linting rules recently. Not sure how this happened.

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**